src/components/calculator/Calculator.jsx

```diff
@@ -1,67 +1,57 @@
-import { PureComponent } from "react";
+import { useState } from "react";
 import "./Calculator.css";
 import calculate from "../../logic/calculate";
-class Calculator extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-  }
+const Calculator = () => {
+  const [calc, setCalc] = useState({ total: 0, next: null, operation: null });
+  const doCalculate = (e) => {
+    let calculations = calculate(calc, e.target.innerText);
+    setCalc({
+      total: calculations.total,
+      next: calculations.next,
+      operation: calculations.operation,
+    });
+  };
 
```

Comment:
- [ ] your app works as expected however you shouldn't ignore the jsx problems in your app because it ensures quality and clean code so please try to remove those errors below 
<img width="1440" alt="Screenshot 2022-12-02 at 15 51 17" src="https://user-images.githubusercontent.com/61790587/205320433-40bfd6b2-0705-482c-8e23-d68ed5c62404.png">

Reply:
@youmari Hello, and thanks to your review, this error shows up only when I first time run the project, I ran the command `npx eslint .` multiple times and it's returning zero errors, do you have any idea about this problem?

Reply:
@z4cope then  try to fix it with `npx eslint . --fix` and push the code if not  try to do it manually